### PR TITLE
OWNERS: OTA may now approve all graph changes

### PR DESCRIPTION
**Pull request by sdodson**

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -1,18 +1,11 @@
 # See the OWNERS_ALIASES docs: https://git.k8s.io/community/contributors/guide/owners.md#OWNERS_ALIASES
 
 aliases:
+  cincinnati-graph-data-reviewers:
   cincinnati-graph-data-approvers:
-    - crawford
```
Comment (sdodson) [inline OWNERS_ALIASES:5]: @crawford Don't know if you wanted to be left in review/approval for graph changes or not. PTAL

**Comment by eparis** [comment]
I guess you can kick us out, but I'd be fine with just adding yourselves in...

**Comment by wking** [comment]
> I guess you can kick us out, but I'd be fine with just adding yourselves in...

Release-admins have been root approvers [since forever][1], so we're not kicking you out ;).

[1]: https://github.com/openshift/cincinnati-graph-data/commit/efc53392baa32c9e90d3dc83d66443a57815b681#diff-37ffc11190043a72e49391017a305d3aR6

**Pull request merged**